website/integrations: add ssh active user filter to sssd integration doc (#15203)
goauthentik/authentik: website/integrations/services/sssd/index.md

>
> -- https://sssd.io/

> Note that authentik supports _only_ user and group objects. As a consequence, it cannot be used to provide automount or sudo configuration nor can it provide netgroups or services to `nss`. Kerberos is also not supported.

Reviewer: ```suggestion
Note that authentik supports _only_ user and group objects. As a result, it cannot be used to provide automount or sudo configuration, nor can it supply netgroups or services to `nss`. Kerberos is also not supported.
```

> Note that authentik supports _only_ user and group objects. As a consequence, it cannot be used to provide automount or sudo configuration nor can it provide netgroups or services to `nss`. Kerberos is also not supported.

Author: I think it' authentik in this case

## Preparation

The following placeholders are used in this guide:

- `authentik.company` is the FQDN of the authentik LDAP outpost installation.
- `ldap.baseDN` is the Base DN you configure in the LDAP provider.

> - `ldap.domain` is (typically) an FQDN for your domain. Usually it is just the components of your base DN. For example, if `ldap.baseDN` is `dc=ldap,dc=goauthentik,dc=io` then the domain might be `ldap.goauthentik.io`.

Reviewer: ```suggestion
- `ldap.domain` is typically a fully qualified domain name (FQDN) representing your domain. It’s often derived from the components of your base DN. For example, if `ldap.baseDN` is `dc=ldap,dc=goauthentik,dc=io`, then the domain would be `ldap.goauthentik.io`.
```

> - `ldap.searchGroup` is the "Search Group" that can can see all users and groups in authentik.

Reviewer: ```suggestion
- `ldap.searchGroup` refers to the "Search Group" that has permission to view all users and groups within authentik.
```

## sssd configuration

First, install the necessary sssd packages on your host. Very likely the package is just `sssd`.

:::note

> This guide well help you configure the `sssd.conf` for LDAP only. You will likely need to perform other tasks for a usable setup like setting up automounted or autocreated home directories that are beyond the scope of this guide. See the "additional resources" section for some help.

Reviewer 2: ```suggestion
This guide well help you configure the `sssd.conf` for LDAP only. You will likely need to perform other tasks for a usable setup like setting up auto-mounted or auto-created home directories that are beyond the scope of this guide. See the "additional resources" section for some help.
```

ldap_default_authtok = ${sssd.serviceAccountToken}
```

> You should now be able to start sssd; however, the system may not yet be setup to use it. Depending on your platform, you may need to use `authconfig` or `pam-auth-update` to configure your system. See the additional resources section for details.

Reviewer 2: ```suggestion
You should now be able to start sssd; however, the system may not yet be set up to use it. Depending on your platform, you might need to use `authconfig` or `pam-auth-update` to configure your system. See the additional resources section for details.
```
